tests/jsunit/field_variable_test.js

```diff
@@ -146,7 +148,8 @@ function test_fieldVariable_dropdownVariableAndTypeDoesNotExist() {
   var result_options = Blockly.FieldVariable.dropdownCreate.call(
```

Comment:
Include a test with the old JSON options as well. [''] is a reasonable default so there's no reason to break backwards compatibility.